content/english/_index.md

```diff
@@ -3,25 +3,29 @@
+  - title: "Quality as Paramount"
+    image: "/images/service-1.png"
+    content: "In our projects, we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget --- you can trust us that your product will fly in no time. We focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals leverages both practical and abstract strategies to create reliable software solutions that meet our clients' requirements. We value partnerships with the open-source community, fostering an environment where diverse perspectives converge to shape tailored solutions that best fit the product."
     button:
       enable: true
       label: "Get to know Us"
       link: "team"
 
-  - title: "Efficiency at the best Price"
-    image: "/images/service-1.png"
-    content: "Welcome to Don't rely on Nulls, where we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget. As a boutique, we focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals values both practical and abstract strategies to create reliable software solutions that meet our clients' requirements."
+  - title: "Humility and Learning"
+    image: "/images/service-2.png"
+    content: "Integrity is paramount to our approach. We adhere to the highest ethical standards and craftsmanship to ensure our solutions align seamlessly with our clients' objectives. In a rapidly evolving tech landscape, we remain humble learners, valuing grounded ideas and the pursuit of improvement. Our team embraces ongoing education and exploration to shape the future of software development. Collaboration is at the heart of our ethos. Our hobby projects demonstrate how dedicated we are into exploration and facing uncharged territories."
+    button:
+      enable: true
+      label: "Check Projects"
+      link: "projects"
 
   - title: "Business Value via FP"
     image: "/images/service-3.png"
-    content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets philosophical reflection, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty and elegant enough that you will want to hang on your wall."
+    content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets from-first-principles reflection, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty, and maintanable enough that you will like to frame it."
```

Comment:
```suggestion
    content: "Choose Don't rely on Nulls for dependable software solutions. Join us on a journey where practical innovation meets from-first-principles reflection, guided by our expertise in functional programming, towards a brighter digital horizon. The outcome will be pretty, and maintanable enough that you'd like to frame it."
```

The "will" there feels rather forceful, more like a command than a statement of fact. You can drop the contraction and say "would", but that doesn't read as smoothly.